phabfive/maniphest.py

```diff
@@ -14,14 +17,92 @@
 import yaml
 from jinja2 import Template
 
-
 log = logging.getLogger(__name__)
 
-
 class Maniphest(Phabfive):
     def __init__(self):
         super(Maniphest, self).__init__()
 
+    def task_search(self, created_after=None, updated_after=None, project=None):
+        """
+        Search for Phabricator Maniphest tasks with given parameters.
+
+        Parameters
+        ----------
+        created_after (int, optional): Number of days ago the task was created.
+        updated_after (int, optional): Number of days ago the task was updated.
+        project (str, required): Project name.
+        """
+
+        if not project:
```

Comment:
Since `project` is now not optional anymore we need to change the function signature to reflect this.

1. Non-optional arguments comes before optional ones.
2. The function signature should not have a default (currently `None`) to reflect that this needs to be passed to the function.